app/update.go

```diff
@@ -555,3 +556,27 @@ func (m *Model) toggleKeysEnabled(v bool) {
 	m.keys.selectSingle.SetEnabled(v)
 	m.keys.clearSelected.SetEnabled(v)
 }
+
+func (m *Model) toggleConfirmationKeys(v bool) {
+	m.list.KeyMap.CursorUp.SetEnabled(v)
+	m.list.KeyMap.CursorDown.SetEnabled(v)
+	m.list.KeyMap.Filter.SetEnabled(v)
+	m.list.KeyMap.GoToEnd.SetEnabled(v)
+	m.list.KeyMap.GoToStart.SetEnabled(v)
+	m.list.KeyMap.Quit.SetEnabled(v)
+	m.list.KeyMap.NextPage.SetEnabled(v)
+	m.list.KeyMap.PrevPage.SetEnabled(v)
+	m.list.KeyMap.ShowFullHelp.SetEnabled(v)
+
+	m.keys.remove.SetEnabled(v)
+	m.keys.togglePin.SetEnabled(v)
+	m.keys.togglePinned.SetEnabled(v)
+	m.keys.selectDown.SetEnabled(v)
+	m.keys.selectUp.SetEnabled(v)
+	m.keys.selectSingle.SetEnabled(v)
+	m.keys.clearSelected.SetEnabled(v)
+}
+
+func (m *Model) toggleQuitEnabled(v bool) {
```

Comment:
Why calling it toggle when you pass the value?

A toggle would have been something like


```suggestion
func (m *Model) toggleQuitEnabled() {
previousState :=         m.list.KeyMap.Quit.GetState() // pseudo code here
        m.list.KeyMap.Quit.SetEnabled(!previousState)
```

Reply:
@ccoVeille agreed, originally I had it set up as a toggle but didn't update the name after changing. Will update the code for the next commit :+1: